src/managers/GuildMemberManager.js

```diff
@@ -223,6 +223,19 @@ class GuildMemberManager extends CachedManager {
     return data.reduce((col, member) => col.set(member.user.id, this._add(member, cache)), new Collection());
   }
 
+  /**
+   * The data for editing a guild member.
+   * @typedef {Object} GuildMemberEditData
+   * @property {?string} [nick] The nickname to set for the member
+   * @property {Collection<Snowflake, Role>|RoleResolvable[]} [roles] The roles or role ids to apply
+   * @property {boolean} [mute] Whether or not the member should be muted
+   * @property {boolean} [deaf] Whether or not the member should be deafened
+   * @property {GuildVoiceChannelResolvable|null} [channel] Channel to move the member to
+   * (if they are connected to voice), or `null` if you want to disconnect them from voice
+   * @property {Date|number|null} [communicationDisabledUntil] The date, timestamp or time in seconds
```

Comment:
This should be applied to the `GuildMemberEditData` within the typings.

Reply:
Added with b13faaedf58dfec6f28751077cd1b53a0ec0ec63!

Reply:
Oxford comma?
```suggestion
   * @property {Date|number|null} [communicationDisabledUntil] The date, timestamp, or time in seconds
```

Reply:
I was referring to the interface `GuildMemberEditData`

Reply:
yeah, I don't know why I commented on your review instead of making my own 😅